### PR TITLE
INT-32 (React Static/Dynamic Router)

### PR DESCRIPTION
**Pull request by cchanyu**
# Notes:
- Added static router at the top, for login and home(root)
- Added dynamic router to ProjectListItem components
- Project title is clickable and it redirects to projects page
- User name is clickable and it redirects to users page

### CODE REVIEW

File: src/components/Account/ProjectDisplay.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const ProjectDisplay = (props) => {
```
Comment (richsong) [inline src/components/Account/ProjectDisplay.js]: Let's rename this to ProjectDetailView

File: src/components/Account/UserDisplay.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const UserDisplay = (props) => {
```
Comment (richsong) [inline src/components/Account/UserDisplay.js]: maybe UserDetailView

File: src/index.js [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+import Project from './components/Account/ProjectDisplay';
+import User from './components/Account/UserDisplay';
 import App from './App.js';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const routing = (
+  <Router>
+    <Navbar>
+      <App />
+    </Navbar>
```
Comment (richsong) [inline src/index.js]: Does this seem right to you? Our app component is within the navbar?

Comment (cchanyu) [reply src/index.js]: When I do
<Navbar />
<App />
Router doesn't function.

**Pull request merged**